types/node-fetch/index.d.ts

```diff
@@ -1,11 +1,11 @@
 /// <reference types="node" />
+// Minimum TypeScript Version: 4.7
```

Comment:
No need for this comment, package.json is the canonical place.

Reply:
Will remove! [This part of the readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md?plain=1#L542) says to add it, which is why I did so

Reply:
Ugh, more leftovers.